src/top_level_parser.rs

```diff
@@ -65,6 +65,14 @@ where
             healthy: true,
             targets: vec![],
         })),
+        (string("LBUG").map(|_| HealthCheckStat {
+            healthy: false,
+            targets: vec![],
+        })),
+        (string("NOT HEALTHY").map(|_| HealthCheckStat {
```

Comment:
Nits: 

- This should be a const since we repeat it below
- We should condense these two blocks with an [`or`](https://docs.rs/combine/latest/combine/trait.Parser.html#method.or)